SU-HSDO/suhumsci
# Shs 5 embed bug fixes

## Summary
* This fixes a Twitter embed bug where the Tweet's height would be cut off.
* This also fixes the ISSUU embed width to span the entire container.

## Urgency
_'low_

## Screenshot
![Screen Shot 2022-05-04 at 2 03 57 PM](https://user-images.githubusercontent.com/8405274/166772493-82191638-6ac9-4de8-8298-8bb302b336f9.png)

![Screen Shot 2022-05-04 at 2 07 03 PM](https://user-images.githubusercontent.com/8405274/166780320-8b080d03-187c-48ae-9857-32a968cbba09.png)


## Steps to Test
1. Pull down the branch, run `npm run watch` in the main theme folder (`humsci_basic`), clear the cache and visit `/qa-embeds` on your local. 
2. Verify the embedded tweet is no longer cut off.
3. Verify the ISSUU embed spans the width of the container.
